File: preview/understand/intent_actions/update-quickstart-example-1/update-quickstart-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Preview.Understand.Assistant.Intent;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var intentActions = IntentActionsResource.Update(
            actions: "{[{\"say\": \"I was going to look for my missing watch, but I could never find the time.\"}]}",
            pathAssistantSid: "UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathIntentSid: "tell-a-joke"
        );

        Console.WriteLine(intentActions.AssistantSid);
    }
}

File: preview/understand/intent_actions/update-quickstart-example-1/update-quickstart-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

intent_actions = @client.preview
  .understand
  .assistants('UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .intents('tell-a-joke')
  .intent_actions
  .update(
     actions: '{[{"say": "I was going to look for my missing watch, but I could never find the time."}]}'
   )

puts intent_actions.assistant_sid
